proposals/0002-remove-currying.md

```diff
@@ -3,7 +3,7 @@
 * Proposal: [SE-0002](0002-remove-currying.md)
 * Author: [Joe Groff](https://github.com/jckarter)
 * Status: **Implemented (Swift 3)**
-* Commit: [apple/swift@983a674](https://github.com/apple/swift/commit/983a674e0ca35a85532d70a3eb61e71a6d024108)
+* Implementation: [apple/swift@983a674](https://github.com/apple/swift/commit/983a674e0ca35a85532d70a3eb61e71a6d024108)
```

Comment:
This is an interesting one.  I think there is a difference between the "Implementation" that was used to evaluate the proposal and the final "commit" where the change was integrated into `master`.  Even once there is an implementation that is acceptable for reviewing a proposal it may not be in sufficient stage to just merge into `master`.

Reply:
I added those commit links (to proposals without SR bugs) in case they'd be useful to someone implementing a similar feature.

The original commit for SE-0002 might be apple/swift@30af42fda98fa1aae81ca00624cafb2090143f5b but that's not very useful either.

I can remove the commit links, and/or close this pull request (if you prefer the existing proposal template).

Reply:
I think the links are useful.  I'm more just thinking about terminology and what they convey.  After thinking about this some more, I think this is a good change as is.